stable/pipelines/values.yaml

```diff
@@ -836,6 +840,13 @@ pipelines:
       metrics:
         enabled: {{ .Values.pipelines.metrics.enabled }}
       logging:
+
+        view:
+          enabled: {{ .Values.pipelines.logging.view.enabled }}
+          refreshRate: 10s
+          concurrentSessionsPerUser: 10
+          files: "router-service.log,router-request.log,marshaller.log,extensionSync.log,hookHandler.log,pipelineSync.log,runTrigger.log,templateSync.log,logup.log,api.log,stepTrigger.log,cron.log,www.log"
```

Comment:
should this be added here? By default, in an on-prem install, users should be able to look at any log files right?

Reply:
@rahulsadanandan No need to add files entry here. Pls remove.

Reply:
Its removed